plugins/turing/config/config.go

```diff
@@ -1,22 +1,23 @@
 package config
 
 import (
+	"fmt"
 	"sync"
 
-	"github.com/caraml-dev/turing/engines/experiment/pkg/request"
 	"github.com/go-playground/validator/v10"
 )
 
-// RequestParameter captures a single parameter's config for parsing the incoming
-// request and extracting values
-type RequestParameter struct {
```

Comment:
This struct is almost exactly the same as `type Variable` below. So, replaced usage with that instead.

But, this change will affect the parameter names set by the experiment manager / used by the experiment runner (field `parameter` -> `name`). Therefore, we would need to update the plugin version to latest, for all existing records in the DB, to support redeploy of existing routers with new experiment manager.